labellab-server/config/dbURI.js

```diff
@@ -2,15 +2,19 @@
 require('dotenv').config()
 
 var dbHost = process.env.DB_HOST || 'localhost'
-var dbName = process.env.DB_NAME
+var dbName = process.env.DB_NAME 
 var dbCluster = process.env.DB_CLUSTER
 var dbUser = process.env.DB_USERNAME
 var dbPass = process.env.DB_PASSWORD
+var dbPort = process.env.DB_PORT
 
 let mongoURI
 
 if(dbName && dbUser && dbPass){
-	mongoURI = '' + dbHost + '://'+ dbUser + ':' + dbPass + '@' + dbCluster + '/' + dbName
+	mongoURI = 'mongodb+srv://adi:adi9891255973@adi-ocz5j.mongodb.net/test?retryWrites=true&w=majority'
```

Comment:
We have created `.env` for saving credentials, you don't have to hard code your credentials.